code/game/machinery/computer/depot_consoles.dm

```diff
@@ -508,9 +500,12 @@
 	if(!portal_enabled && myportal)
 		to_chat(user, "<span class='notice'>Outgoing Teleport Portal: deactivating... please wait...</span>")
 		return
-	toggle_portal()
+
+	var/turf/tele_target = choosetarget()
+	if(!tele_target)
+		return
+	toggle_portal(tele_target)
```

Comment:
`toggle_portal` doesnt take any args. I think you meant to update it on line 511 to accept this as an arg and pass it through to `update_portal` on line 513

Reply:
Updated in next commit